packages/store/src/releasesSlice/thunks.ts

```diff
@@ -275,8 +275,8 @@ export const removeReleases = createAsyncThunk(`${sliceName}/removeReleases`, (r
   return Promise.all(deleteRequests);
 });
 
-export const selectRelease = createAsyncThunk(`${sliceName}/selectRelease`, (release, { dispatch }) => {
-  const name = release ? release.name || release : null;
+export const selectRelease = createAppAsyncThunk(`${sliceName}/selectRelease`, (release: Release | string | null, { dispatch }) => {
+  const name = release && typeof release === 'object' ? release.name : release;
```

Comment:
That's not the same - if release is `undefined` it won't be set to null and once again we won't know if this will cause an issue when introducing the package.